app/controllers/dashboard_controller.rb

```diff
@@ -11,10 +11,9 @@ def index
     redirect_to :action => 'show'
   end
 
-  skip_before_action :set_csp_header, :only => :iframe # FIXME: only frame-src
-  skip_before_action :set_x_frame_options_header, :only => :iframe
-
   def iframe
+    override_content_security_policy_directives(:frame_src => ['*'])
```

Comment:
NOTE: This is needed because we are embedded in other product / others are embedded in us.